release-content/0.15/release-notes/14257_Pack_multiple_vertex_and_index_arrays_together_into_growab.md

```diff
@@ -2,3 +2,17 @@
 <!-- https://github.com/bevyengine/bevy/pull/14257 -->
 
 <!-- TODO -->
+
+**Bevy 0.15** changes the way meshes are uploaded to the GPU to greatly improve performance. Instead of submitting vertex and index buffers for every mesh individually as in Bevy 0.14, now they are coalesced into 'slabs' of configurable size. This cuts down on wgpu per-buffer overhead, winning us up to 2x speedups!
```

Comment:
```suggestion
**Bevy 0.15** changes the way meshes are stored on the GPU to greatly improve performance. Instead of separate vertex and index buffers for every mesh individually as done in Bevy 0.14, now they are coalesced into 'slabs' of configurable size. This cuts down on how frequently we need to change bind groups, winning us up to 2x speedups!
```

State changes are right now a much bigger deal than upload speed or buffer creation (which we've barely optimized).

Reply:
We should also mention it's specifically a CPU performance win, and not so much (that I know of?) GPU win.

Reply:
It could be due to https://gpuopen.com/learn/understanding-gpu-context-rolls, but in our case CPU performance is a way bigger bottleneck that none of this stuff matters.